apps/explorer/src/pages/transaction-result/programmable-transaction-view/__test__/Command.test.tsx

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { describe, expect, it } from 'vitest';
+
+import {
+    Command,
+    CommandType,
+} from '~/pages/transaction-result/programmable-transaction-view/Command';
+
+describe('Command', () => {
```

Comment:
tbh I'd be fine if this test didn't exist given this is mostly just a validation of the other tests for formatting commands that we already have